xbmc/video/windows/GUIWindowVideoNav.cpp

```diff
@@ -845,8 +845,9 @@ void CGUIWindowVideoNav::OnDeleteItem(CFileItemPtr pItem)
   else if (m_vecItems->GetContent() == "tags" && pItem->m_bIsFolder)
   {
     CGUIDialogYesNo* pDialog = CServiceBroker::GetGUI()->GetWindowManager().GetWindow<CGUIDialogYesNo>(WINDOW_DIALOG_YES_NO);
-    pDialog->SetHeading(CVariant{432});
-    pDialog->SetLine(1, CVariant{ StringUtils::Format(g_localizeStrings.Get(433).c_str(), pItem->GetLabel().c_str()) });
+    pDialog->SetHeading(CVariant{10058});
+    pDialog->SetLine(1, CVariant{StringUtils::Format(g_localizeStrings.Get(10059).c_str(),
+                                                     pItem->GetLabel().c_str())});
     pDialog->SetLine(2, CVariant{""});
```

Comment:
By handling "tag" removeal in CGUIDialogVideoInfo::DeleteVideoItemFromDatabase, this entire `else if` can be removed and the final `else` in calling CGUIDialogVideoInfo::DeleteVideoItem will do the delete.
Better to have `<delete>` and CONTEXT_BUTTON_DELETE actions use common processing.

Reply:
Done.